src/vue/backend/src/views/guided-setup/Omnibar.vue

```diff
@@ -0,0 +1,66 @@
+<template>
```

Comment:
Out of scope. This file should be done in http://jira.hq.idx.org/browse/UXG-1269

Remove Test code